encryption/src/pkcs1_c/system.py

```
@@ -421,6 +432,37 @@ def load_public_key_pem(self, public_key_pem):
         # returns the keys tuple
         return keys
 
+    def load_certificate_pem(self, certificate_pem):
+        # matches the certificate header/footer token in case no match
+        # is done raises an exception indicating the problem
+        certificate_pem_match = CERTIFICATE_VALUE_REGEX.match(certificate_pem)
+        if not certificate_pem_match:
+            raise exceptions.InvalidFormatException(
+                "certificate header/footer not found"
+            )
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)